app/lib/config.sh

```diff
@@ -56,7 +56,9 @@ config_load_envs() {
 	[ -d "$envs" ] || return 0
 	for file in "$envs"/*; do
 		name="$(basename "$file")"
-		export "$name=$(config_get "environment/$name")"
+		config=$(config_get "environment/$name")
+		substitued_config=$(envsubst | "$config")
```

Comment:
I think this needs to be the other way around and prefixed with an echo like this:

```bash
		substitued_config=$(echo "$config" | envsubst)
```

otherwise it will try to execute the value of the config key as a command. Then you could just simplify the change to:

```bash
export "$name=$(config_get "environment/$name" | envsubst)"
```